campari/tests/test_config.yaml

```diff
@@ -0,0 +1,52 @@
+photometry:
+  campari:
+    cutout_size: 11
+    initial_flux_guess: 3000
+    fetch_SED: true
+    source_phot_ops: true
+    use_roman: true
+    weighting: true
+    make_initial_guess: true
+    method: lsqr
+    pixel: false
+    subtract_background: true
+    use_real_images: true
+
+    paths:
+      roman_path: /hpc/group/cosmology/OpenUniverse2024
+      sn_path: /hpc/group/cosmology/OpenUniverse2024/roman_rubin_cats_v1.1.2_faint/
+
+    grid_options:
+      type: contour
+      percentiles:
+      - 0
+      - 90
+      - 98
+      - 100
```

Comment:
I would have naively thought these would have been indented by an additional two space, by I am not a YAML expert.